persistence/jdbc/src/main/java/org/infinispan/persistence/jdbc/DatabaseType.java

```diff
@@ -7,9 +7,9 @@
  * @since 4.1
  */
 public enum DatabaseType {
-   MARIA_DB, MYSQL, POSTGRES, DERBY, HSQL, H2, SQLITE,
+   MARIA_DB, MYSQL, MARIADB, POSTGRES, DERBY, HSQL, H2, SQLITE,
```

Comment:
Why do we have two enum values for MariaDB?

Reply:
I added this new value but the logic that I created is no longer being used, this value can be removed.

Reply:
I have reformatted this class to remove the duplicate, put each db on a single line and sort them